bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -81,7 +83,30 @@ export default class Whiteboard extends React.Component {
     }
   }
 
+  renderSlideControlsContainer() {
+    if (this.props.currentSlide) {
+      return (
+        <SlideControlsContainer
+          currentSlideNum={this.props.currentSlide.slide.num}
+          presentationId={this.props.currentSlide.presentationId}
+        />
+      );
+    } else {
+      return null;
+    }
+  }
+
   render() {
+    if (this.props.currentSlide) {
+      slideControlsContainer =
+        <SlideControlsContainer
+          currentSlideNum={this.props.currentSlide.slide.num}
+          presentationId={this.props.currentSlide.presentationId}
+        />;
+    } else {
+      slideControlsContainer = null;
+    }
```

Comment:
It looks like this whole if / else block in render() is not needed.
Your renderSlideControlsContainer() func returns either null or the <SlideControlsContainer> component depending whether you have a current slide or not.
This check generally does nothing, you don't use slideControlsContainer variable anywhere.